guides/common/modules/proc_applying-errata-to-hosts.adoc

```diff
@@ -1,49 +1,26 @@
 [id="applying-errata-to-hosts"]
 = Applying errata to hosts
 
-You can use the API to apply errata to a host, host group, or host collection.
-The following is the basic syntax of a PUT request:
-
-[options="nowrap", subs="+quotes,attributes"]
-----
-$ curl --header "Accept:application/json" \
---header "Content-Type:application/json" --request PUT \
---user _My_User_Name_:__My_Password__ \
---data _My_JSON_Formatted_Data_ https://_{foreman-example-com}_
-----
-
-You can browse the built-in API doc to find a URL to use for applying errata.
-You can use the {ProjectWebUI} to help discover the format for the search query.
-Navigate to *Hosts* > *Host Collections* and select a host collection.
-Go to *Collection Actions* > *Errata Installation* and notice the search query box contents.
-For example, for a Host Collection called _my-collection_, the search box contains `host_collection="my-collection"`.
-
-[id="exam-API_Guide-Applying_Errata_to_a_Host"]
-.Applying errata to a host
-
-This example uses the API URL for bulk actions `/katello/api/hosts/bulk/install_content` to show the format required for a simple search.
-
-Example request:
+You can use {Project} apply errata to hosts.
+ifndef::rest-api[]
+To use the API instead of the {ProjectWebUI}, see the xref:api-applying-errata-to-hosts[].
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Hosts* > *Host Collections*.
+. Select a host collection.
```

Comment:
```suggestion
. Click a host collection to view the *Details* tab.
```
If the docs say "Select" whenever the user clicks to view details, you can ignore this comment for the sake of consistency.